pkg/pillar/build-rstats.yml

```diff
@@ -0,0 +1,29 @@
+# linuxkit build template
+#
+# Copyright (c) 2018 Zededa, Inc.
+# SPDX-License-Identifier: Apache-2.0
+#
+# this file should be kept consistence with build.yml
```

Comment:
Perhaps add this line instead into build.yml ...

Reply:
remove it.